axon-framework/spring-boot-integration.md

```diff
@@ -32,36 +32,78 @@ To register Axon's JPA Entities, include the relevant packages, as described bel
 ## Demystifying Axon Spring Boot Starter
 
 With a lot of things happening in the background, it sometimes becomes difficult to understand how an annotation or just
-including a dependency enables so many features.
+including a dependency enables so many features. The way `@AutoConfiguration` classes need to be configured has changed from Spring Boot 2 to Spring Boot 3.
```

Comment:
After some one-on-one, we noticed that mentioning this `@AutoConfiguration` annotation adjustments in Spring Boot 2.7 is not mandatory to mention. Furthermore, the code tabs already show there's a difference between Spring Boot 2 and 3 on this subject.